src/confdb/confdb.h

```diff
@@ -27,6 +27,9 @@
 #include <tevent.h>
 #include <ldb.h>
 #include <ldb_errors.h>
+#ifndef gid_t
+#include <unistd.h>
```

Comment:
On July 14, 2016 at 1:42:45 pm +02:00, lslebodn <<notifications@github.com>> wrote:

> In src/confdb/confdb.h <https://github.com/SSSD/sssd/pull/4#discussion_r70791085>:
> > @@ -27,6 +27,9 @@
> > #include <tevent.h>
> > #include <ldb.h>
> > #include <ldb_errors.h>
> > +#ifndef gid_t
> > +#include <unistd.h>
> 
> man sys_types.h says that gid_t shoudl be defined in sys/types.h
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub <https://github.com/SSSD/sssd/pull/4/files/bd4c5186e85619c0d6fdc36690b311734e552fe4#r70791085>, or mute the thread <https://github.com/notifications/unsubscribe/AAipUnYbI2psUazPmNphNqHjRAFMVvuFks5qViC1gaJpZM4JMLmi>.
> 
> 
> 
> 
> It isn't defined there on musl, and tho I don't know why musl haven't put it there, my guess is that they thought that unistd.h is the more appropriate place for that.

Mvh
Daniel Kuehn, 073-181 577 2
Data@UrService
Hemsida: <http://www.dataaturservice.se>